packages/jsii/lib/symbol-id.ts

```diff
@@ -5,6 +5,8 @@ import * as ts from 'typescript';
 export function symbolIdentifier(
   typeChecker: ts.TypeChecker,
   sym: ts.Symbol,
+  valueAsIs = true,
```

Comment:
Can we think of a better name for this argument?

Perhaps it should go into an `options` struct as well...